cpp/src/io/json/reader_impl.hpp

```diff
@@ -181,7 +185,8 @@ class reader::impl {
   explicit impl(std::unique_ptr<datasource> source,
                 std::string filepath,
                 json_reader_options const &options,
-                rmm::mr::device_memory_resource *mr);
+                rmm::mr::device_memory_resource *mr,
```

Comment:
Nitpick: Was it intentional not to switch the order of `mr` and `stream` here?

Reply:
Nope, just missed that one somehow. Thanks for pointing it out!